spec/services/top_region_service_spec.rb

```diff
@@ -16,6 +16,34 @@ def refresh_views
     end
   end
 
+  it "has a call convienence method" do
```

Comment:
[nit] This spec also seems to be validating some data, could you update the `it` description.